front/pages/api/w/[wId]/assistant/conversations/[cId]/index.ts

```diff
@@ -19,6 +19,7 @@ export const PatchConversationsRequestBodySchema = t.type({
     t.literal("unlisted"),
```

Comment:
What is unlisted? I don’t remember where it’s used. Shouldn’t we use that if it’s here?

Reply:
Didn't remember either. I checked, it's used as default. Most our convos are "unlisted", IMHO meant to be "shareable but not discoverable" at a time where we wanted a "workspace conversations" tab where everybody could see others' publicly shared convos

Reply:
I think it's good that we have a specific value for those "test" conversations, so in the future we can easily distinguish them (either have a tab with only those, or delete only those, or etc.)

in any case it willl be quite easy to change if we want, only an sql update where vis = test, no complex migration

=> going to move forward with the merge as-is (this afternoon), but remaining open to change in any other way

Reply:
FYI the unused value for convo vis is 'workspace', no workspace convo since Aug 10 in the dust workspace